experimenter/experimenter/theme/static_src/rollup.config.mjs

```diff
@@ -0,0 +1,19 @@
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import terser from '@rollup/plugin-terser';
+
+
+export default { 
```

Comment:
This is formatted funky. Can you run prettier on this? `npx prettier -w rollup.config.mjs`